README.Rmd

```diff
@@ -112,12 +119,22 @@ res <- arc_raster(
 terra::plotRGB(res, 4, 3, 2, scale = max(landsat[["maxValues"]]))
 ```
 
-## Authorization
+## Authorization and publication
+
+### Authorization for non-public data access
+
+Authorization is required for users who wish to access data that is in a non-public ArGIS Online (AGOL) organization, in an ArcGIS Enterprise portal, or wish to upload data to or modify data in AGOL.
+
+For more information on code based authorization using the `auth_code()` function or client authorization using `auth_client()` see the `{arcgislayers}` [authorization article](articles/Authorization.html).
+
```

Comment:
The article was removed from the package in favor of the article on the site to dedupe docs.
```suggestion
For more information on code based authorization using the `auth_code()` function or client authorization using `auth_client()` see the [authorization article](https://r.esri.com/r-bridge-site/location-services/connecting-to-a-portal.html).
```